solana/programs/01_hello_world/Cargo.toml

```
@@ -20,7 +20,7 @@ cpi = ["no-entrypoint"]
 idl-build = ["anchor-lang/idl-build", "wormhole-anchor-sdk/idl-build"]
 
 [dependencies]
-solana-program = "1.18.17"
+solana-program = "=2.1.0"
```

nit: Maybe keep dependency versions consistent across programs
```suggestion
solana-program = "2.1.0"
```